File: 0280-wiggle-sort/0280-wiggle-sort.js
/**
 * @param {number[]} nums
 * @return {void} Do not return anything, modify nums in-place instead.
 */
var wiggleSort = function(nums) {
    nums.sort((a,b)=>a-b);
    let arr = [...nums];
    let left = 0;
    let right = nums.length-1;
    
    for(let i=0; i<nums.length; i++){
        if(i % 2 === 0) nums[i] = arr[left++];
        else nums[i] = arr[right--];
    }
};

File: 0280-wiggle-sort/0280-wiggle-sort.java
class Solution {
    public void wiggleSort(int[] nums) {
        Arrays.sort(nums);
        int [] arr = nums.clone();
        int left = 0;
        int right = nums.length-1;
        for(int i=0; i<nums.length; i++){
            if(i % 2 ==0) nums[i] = arr[left++];
            else nums[i] = arr[right--];
        }
    }
}